src/bosh-director-core/spec/bosh/director/core/templates/job_template_loader_spec.rb

```diff
@@ -128,29 +131,32 @@ module Bosh::Director::Core::Templates
       end
 
       it 'includes only monit erb object when no other templates exist' do
-        tarball_path = create_job_tarball('release-job-no-templates', 'monit file erb contents', tmp_file, {})
+        tarball_path = create_job_tarball('plip-job-name', 'monit file erb contents', tmp_file, {})
```

Comment:
There has been a reason indeed 🙂. Having different names for `plan-job-name` and `release-job-name` was a defect, that hadn’t show up before because `spec.name` wasn’t used.

There were no reason for distinguishing “plan” or “release” job, because there where designating the very same thing. These two names actually had to be the same. That’s how I started to use “plop“, and then when I needed a different name later on, I’ve used “plip”.

But I get your point. I’ll use `fake-job-name-1` and `fake-job-name-2` instead.